src/groups/mwc/mwcex/mwcex_future.h

```diff
@@ -232,7 +232,7 @@ class Future_Exception {
 
     /// Provides a "small" dummy object which size is used to calculate the
     /// size of the on-stack buffer used for optimization.
-    struct Dummy {
+    struct Dummy : public mwcu::NoOp {
```

Comment:
I kinda get the rational behind changes in `mwcex_job` -- the `Dummy` is involved with a functor, but what is the rational here? Does the code not compile if you leave this dummy (and the one below) as is?